test/updateMessages.test.js

```diff
@@ -0,0 +1,28 @@
+import { ipcRenderer } from 'electron';
```

Comment:
how are these tests run? Does npm run them with `npm test`? or some other npm command?

Reply:
I plan to implement tests like how you guys do (mocha).

These "tests" are more of fixtures I throw into the GUI to see how it visually looks. Like how you did it last year

Reply:
I think I will be adding mocha testing to the branch before fully merging to `dev-2018`, as well as creating a separate folder called `fixtures` and adding that in.

We can change the name `devMode` in the `config.json` to `fixtures` to be more clear about that.

Reply:
Resolved in https://github.com/NGCP/GCS/commit/7281b09d05d28cd92286e87185713d2b03d56da1